ballerina-tests/tests/60_client_send_error_in_client_bidi_streaming.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+service ErrorSendService {
+	rpc sendErrorClientStreaming(stream google.protobuf.StringValue) returns (google.protobuf.Int32Value);
+  rpc sendErrorBidiStreaming(stream google.protobuf.StringValue) returns (stream google.protobuf.Int32Value);
```

Comment:
Indentation?

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/444/commits/379da88fe9706c7fd054608e8b041974a6951308